clang/include/clang/Basic/LangOptions.h

```diff
@@ -237,6 +237,11 @@ class LangOptionsBase {
     /// in the initializers of members of local classes.
     Ver18,
 
+    /// Attempt to be ABI-compatible with code generated by Clang 19.0.x.
+    /// This causes clang to:
+    ///   - Incorrect Mangling of CXXCtorVTable
```

Comment:
```suggestion
    ///   - Incorrectly mangle CXX vtable substitutions in some cases...
```

It would be great if you could flesh out the behavior changes here.

Reply:
Thanks. Updated.